### PR TITLE
Bugfix FXIOS-4317 [v123] Enable dynamic type on history highlight section on Firefox home page

### PR DESCRIPTION
**Pull request by imsanchit**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-4317)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/10896)

### Before PR
Screenshot 2023-12-29 at 7 37 22 PM

### After PR
Screenshot 2023-12-29 at 7 35 29 PM

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

File: firefox-ios/Client/Frontend/Home/HistoryHighlights/HistoryHighlightsCell.swift [hunk 1]
```diff
@@ -131,6 +132,19 @@ class HistoryHighlightsCell: UICollectionViewCell, ReusableCell {
...
+        super.layoutSubviews()
+        let size: CGFloat = 5
+        let distance: CGFloat = 0
+        let rect = CGRect(
+            x: -size,
+            y: contentView.frame.height - (size * 0.4) + distance,
+            width: contentView.frame.width + size * 2,
+            height: size
+        )
+        contentView.layer.shadowPath = UIBezierPath(ovalIn: rect).cgPath
```
Comment (adudenamedruby) [inline firefox-ios/Client/Frontend/Home/HistoryHighlights/HistoryHighlightsCell.swift:145]: Shouldn't this code only be added if it's a cell that needs a shadow?

Comment (imsanchit) [reply firefox-ios/Client/Frontend/Home/HistoryHighlights/HistoryHighlightsCell.swift]: Hey, thanks for pointing it out, have updated the PR with a fix. Please have a look.

**Review by adudenamedruby — APPROVED** [review]

**Comment by adudenamedruby** [comment]
Looks like there's some Swiftlint issues.

**Comment by imsanchit** [comment]
Hey, fix the swiftlint issue.

**Pull request merged**